depthai_bridge/include/depthai_bridge/TrackDetectionConverter.hpp

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include <deque>
+#include <memory>
+#include <string>
+#include <vision_msgs/msg/detection2_d_array.hpp>
```

Comment:
Currently we are following guideline to keep `<>` tags for system libraries, could you change those includes to use quotes?

Reply:
Ok, I've changed it.